Dockerfile

```diff
@@ -22,9 +22,19 @@ COPY --from=jenkinsfilerunner-mvncache /mavenrepo /mavenrepo
 ADD . /jenkinsfile-runner
 RUN cd /jenkinsfile-runner && mvn package
 
-FROM jenkins/jenkins:${JENKINS_VERSION}
+FROM jenkins/jenkins:${JENKINS_VERSION} as jenkins
 USER root
-RUN mkdir /app && unzip /usr/share/jenkins/jenkins.war -d /app/jenkins
+# Delete big files not needed
+RUN mkdir /app && unzip /usr/share/jenkins/jenkins.war -d /app/jenkins && \
+  rm -rf /app/jenkins/scripts /app/jenkins/jsbundles /app/jenkins/css /app/jenkins/images /app/jenkins/help /app/jenkins/WEB-INF/detached-plugins /app/jenkins/winstone.jar /app/jenkins/WEB-INF/jenkins-cli.jar /app/jenkins/WEB-INF/lib/jna-4.5.2.jar
+
+FROM openjdk:8-jdk
```

Comment:
How will this work when the specified JENKINS_VERSION (which is a parameter) is modified to something needing Java 11?

Reply:
Unfortunately Docker is not that good at multi-platform support as it. We would need to add a new Dockerfile or to somehow implement Dockerfile generation from scripts and then to move the release flow to ci.jenkins.io and Trusted CI

Reply:
Yes, but that situation is working right now because the proper JDK is being set by the Jenkins Docker image itself right?

Reply:
No, JDK comes from the base image which is openjdk:8-jdk